code/modules/projectiles/projectile/energy/net_snare.dm

```diff
@@ -5,16 +5,24 @@
 	damage_type = STAMINA
 	hitsound = 'sound/weapons/taserhit.ogg'
 	range = 10
+	var/obj/item/device/radio/beacon/projtarget
 
 /obj/item/projectile/energy/net/Initialize()
 	. = ..()
 	SpinAnimation()
+	set_projtarget()
+
+/obj/item/projectile/energy/net/proc/set_projtarget()
+	var/obj/item/ammo_casing/energy/net/N = loc
+	if(istype(N))
+		projtarget = N.drag.guntarget
```

Comment:
No way to do this on the ammo_casing when it reloads? Would be more OO to have the casing set the target rather than the projectile asking if there's an ammo_casing for the target.